src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -133,66 +133,66 @@ public boolean hasHTMLTags(String text){
         Map<String, List> aMap = new LinkedHashMap<String, List>();
         aMap.put("golocalprov.com", Arrays.asList(
-                "*[id=slideshow-wrap]"
+                "[id=slideshow-wrap]"
             ));
         aMap.put("cmo.com", Arrays.asList(
-                "*[id=getupdatesform]"
+                "[id=getupdatesform]"
             ));
         aMap.put("bestpaths.com", Arrays.asList(
-                "*[id=secondary]"
+                "[id=secondary]"
             ));
         aMap.put("beet.tv", Arrays.asList(
-                "*[class*=single-recent-post-container]"
+                "[class*=single-recent-post-container]"
             ));
         aMap.put("efytimes.com", Arrays.asList(
-                "*[class=data-para]"
+                ".data-para"
             ));
         aMap.put("wn.com", Arrays.asList(
-                "*[class=caroufredsel_wrapper]"
+                ".caroufredsel_wrapper"
             ));
         aMap.put("www.reuters.com", Arrays.asList(
-                "*[class=section main-content]", // odd case the "section main-content" class actually contains only the title.
+                ".section.main-content", // odd case the "section main-content" class actually contains only the title.
                 "div[id=specialFeature]"         // remove non-article section
             ));
         aMap.put("investors.com", Arrays.asList(
-                "*[class=special-report]",
-                "*[class=more-news]"
+                ".special-report",
+                ".more-news"
             ));
         aMap.put("einnews.com", Arrays.asList(
-                "*[class=headlines mini]"
+                ".headlines.mini"
             ));
         aMap.put("fortune.com", Arrays.asList(
-                "*[id=reprint-modal]"
+                "[id=reprint-modal]"
             ));
         aMap.put("drimble.nl", Arrays.asList(
-                "*[class=dinfoo]",
-                "*[class=dvv]",
-                "*[class=ip]"
+                ".dinfoo",
+                ".dvv",
+                ".ip"
             ));
         aMap.put("americanbanker.com", Arrays.asList(
-                "*[id=whatis-pso-rss-content]"
+                "[id=whatis-pso-rss-content]"
             ));
         aMap.put("schwab.com", Arrays.asList(
-                "*[class=article-disclosure]",
-                "*[class=article-call-to-action]"
+                ".article-disclosure",
+                ".article-call-to-action"
             ));
         aMap.put("theverge.com", Arrays.asList(
-                "*[class=m-linkset__entries-item]",
-                "*[class=m-linkset]",
-                "*[class=\"feature-photos-story feature-photos-column\"]",
-                "*[class*=js-carousel-pane]",
-                "*[id=feature-photos-model]"
+                ".m-linkset__entries-item",
+                ".m-linkset",
+                ".feature-photos-story.feature-photos-column",
+                "[class*=js-carousel-pane]",
```

Comment:
are we intentionally matching on partial class names or would this just be equivalent to `.js-carousel-pane`?